src/algebra/lie/basic.lean

```diff
@@ -944,6 +945,13 @@ by { rw [← mem_coe_submodule, sup_coe_to_submodule, submodule.mem_sup], exact
 lemma eq_bot_iff : N = ⊥ ↔ ∀ (m : M), m ∈ N → m = 0 :=
 by { rw eq_bot_iff, exact iff.rfl, }
 
+lemma top_of_bot_eq_bot : (⊤ : lie_submodule R L ↥(⊥ : lie_submodule R L M)) = ⊥ :=
```

Comment:
Wouldn't you rather prove that *all* submodules of `bot` are equal to each other?

Reply:
Great point; will adjust.